VMS/Resources/src/activitySearch/index.js

```diff
@@ -0,0 +1,23 @@
+﻿
+const filterBox = (className, placeHolder, dropdownParentNode) => {
+    const alwaysShowPlaceholder = (value) => {
+        if (!value.id) {
+            return value.text;
+        }
+        else {
+            return `${placeHolder}`;
+        }
+    }
+    $(document).ready(function () {
```

Comment:
I think that we should remove the Select2 code and library out of our project, since we do not take any advantages of it. We might use it at a later time, but not this time.

Reply:
I removed Select2, fixed size and display of filter box, and added function to "Xóa lọc" button